fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -361,6 +373,7 @@ private fun ReviewQualityInfo(
                         SupportUtils.SumoTopic.HELP,
                     ),
                 )
+                Shopping.bottomSheetReviewQualityExplainerUrlClicked.record()
```

Comment:
This should be in the telemetry middleware, this callback dispatches an action and the telemetry middleware should respond to that action not the ui.